<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>行人追踪检索系统</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="load_image_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>740</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>加载图像</string>
    </property>
   </widget>
   <widget class="QFrame" name="attri_frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>50</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="attri_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>属性检索</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>性别</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>头发长度</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>袖长</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>下装长度</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>下装类型</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>帽子</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_7">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>背包</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_8">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>带包</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>手包</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_10">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>年龄</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_11">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>220</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>上装颜色</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_12">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>260</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>260</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>下装颜色</string>
     </property>
    </widget>
    <zorder>attri_label</zorder>
    <zorder>comboBox_1</zorder>
    <zorder>label_1</zorder>
    <zorder>label_2</zorder>
    <zorder>comboBox_2</zorder>
    <zorder>comboBox_3</zorder>
    <zorder>label_3</zorder>
    <zorder>comboBox_4</zorder>
    <zorder>label_4</zorder>
    <zorder>comboBox_5</zorder>
    <zorder>label_5</zorder>
    <zorder>comboBox_6</zorder>
    <zorder>label_6</zorder>
    <zorder>comboBox_7</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>comboBox_8</zorder>
    <zorder>comboBox_8</zorder>
    <zorder>label_8</zorder>
    <zorder>comboBox_9</zorder>
    <zorder>label_9</zorder>
    <zorder>comboBox_10</zorder>
    <zorder>label_10</zorder>
    <zorder>comboBox_11</zorder>
    <zorder>label_11</zorder>
    <zorder>comboBox_12</zorder>
    <zorder>label_12</zorder>
   </widget>
   <widget class="QFrame" name="result_frame">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>401</width>
      <height>641</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>50</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="origin_video_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>原视频</string>
     </property>
    </widget>
    <widget class="QLabel" name="result_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>检索结果</string>
     </property>
    </widget>
    <widget class="QLabel" name="result_video_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>结果视频</string>
     </property>
    </widget>
    <widget class="Phonon::VideoPlayer" name="origin_videoPlayer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>381</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="Phonon::VideoPlayer" name="result_videoPlayer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>381</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="origin_video_pushButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>290</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>播放</string>
     </property>
    </widget>
    <widget class="QPushButton" name="result_video_pushButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>600</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>播放</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="image_frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>391</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>50</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="image_label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>图像检索</string>
     </property>
    </widget>
    <widget class="QLabel" name="image_show">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>151</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>请加载行人图像</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="load_video_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>740</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>加载视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>740</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始检索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>740</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="attri_checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>使用属性检索</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="image_checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>使用图像检索</string>
    </property>
   </widget>
   <zorder>attri_frame</zorder>
   <zorder>title</zorder>
   <zorder>load_image_button</zorder>
   <zorder>result_frame</zorder>
   <zorder>image_frame</zorder>
   <zorder>load_video_button</zorder>
   <zorder>search_button</zorder>
   <zorder>clear_button</zorder>
   <zorder>attri_checkBox</zorder>
   <zorder>image_checkBox</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
